common/workunit/workunit.hpp

```diff
@@ -1785,4 +1782,17 @@ extern WORKUNIT_API TraceFlags loadTraceFlags(IConstWorkUnit * wu, const std::in
 
 extern WORKUNIT_API bool executeGraphOnLingeringThor(IConstWorkUnit &workunit, unsigned wfid, const char *graphName);
 
+
+class WORKUNIT_API StatisticsAggregator : public CInterface
```

Comment:
see the comment in cpp, re. moving StatisticsAggregator into jstats (and extracting helper functions that need wu)